dags/jetstream.py

```diff
@@ -35,6 +36,23 @@
         dag=dag,
     )
 
+    jetstream_config_changed = GKEPodOperator(
+        task_id="jetstream_run_config_changed",
+        name="jetstream_run_config_changed",
+        image=jetstream_image,
+        email=["ascholtz@mozilla.com", "tdsmith@mozilla.com",],
+        arguments=[
+            "--log_to_bigquery",
+            "rerun-config-changed",
+            "--argo",
+            # the Airflow cluster doesn't have Compute Engine API access so pass in IP 
+            # and certificate in order for the pod to connect to the Kubernetes cluster
+            # running Jetstream
+            "--cluster-ip={{ var.value.jetstream_cluster_ip }}",
+            "--cluster-cert={{ var.value.jetstream_cluster_cert }}"],
+        dag=dag,
+    )
+
```

Comment:
The [entrypoint script was executing two commands](https://github.com/mozilla/jetstream/blob/7fafb7fd71403198319c47a6503cf38d379dd621/bin/entrypoint#L14): run and rerun-config-changed. Splitting them into two separate tasks here